validation-test/stdlib/UnicodeUTFEncoders.swift

```diff
@@ -14,13 +14,18 @@ import StdlibUnittest
 
 import Foundation
 
-protocol TestableUnicodeCodec : UnicodeCodec {
-  associatedtype CodeUnit : FixedWidthInteger
+protocol TestableUnicodeCodec: Unicode.Encoding {
   static func encodingId() -> String.Encoding
   static func name() -> NSString
+
+  // [SE-0163] Test the deprecated `UnicodeCodec` APIs.
+  static func encode(
+    _ input: Unicode.Scalar,
+    into processCodeUnit: (CodeUnit) -> Void
+  )
```

Comment:
This should allow deprecated APIs to be tested, without using the `UnicodeCodec` protocol.